rebasehelper/cli.py

```diff
@@ -163,8 +174,8 @@ class CliHelper(object):
 
     @staticmethod
     def run():
+        debug_log_file = None
         try:
-            debug_log_file = None
```

Comment:
I think catching exception is only for code which can raise an exception. Also you are using `debug_log_file` in one of the exceptions and my IDE marked it that could be referenced before assignment. I know that it's on the first line BUT still I think try: catch: clause should contain code which could raise an Exception and `debug_log_file = None` will not ..